src/components/viz_editor/visualization_editor.py

```diff
@@ -1,55 +1,67 @@
-from .controls.target_select import target_select
-from .controls.age_group_select import age_group_select
-from .controls.uncertainty_select import uncertainty_select
-from .controls.ensemble_select import ensemble_select
-from .controls.models_select import models_select
-from .controls.annotations_input import annotations_input
-from src.components.chart import chart
+from dash import Input, Output, callback, html
+
+from src.components.chart import Chart, ChartControls, PlotType
+
+from .controls import (
+  age_group_select,
+  annotations_input,
+  create_selector_grid_column,
+  location_select,
+  models_select,
+  scenarios_select,
+  target_select,
+  uncertainty_select,
+)
 
 available_rounds = [19]
 current_round = 19
 
 default_control_values = dict(
-  scenarios=['77', '78', '79', '80', '81'],
-  models=['18'],
+  scenarios=['A-2023-10-27', 'B-2023-10-27'],
+  models=['Ensemble_LOP'],
   location='US',
   target='incident_hospitalization',
   age_group='0-130',
   uncertainty='None',
-  ensemble='Ensemble',
   annotations={},
 )
 
+
 def visualization_editor(control_values=None, show_controls=True):
   controls = {**default_control_values, **(control_values or {})}
 
-  init_scenarios = controls['scenarios']
-  init_models = controls['models']
-  init_location = controls['location']
-  init_target = controls['target']
-  init_age_group = controls['age_group']
-  init_uncertainty = controls['uncertainty']
-  init_ensemble = controls['ensemble']
-  init_annotations = controls['annotations']
+  init_scenarios: list[str] = controls.get('scenarios', ['A-2023-10-27', 'B-2023-10-27'])
+  init_models: list[str] = controls.get('models', ['Ensemble_LOP'])
+  init_location: str = controls.get('location', 'US')
+  init_target: str = controls.get('target', 'incident_hospitalization')
+  init_age_group: str = controls.get('age_group', '0-130')
+  init_uncertainty: str | None = controls.get('uncertainty')
+  init_annotations: dict[str, Any] = controls.get('annotations', {})
+  x_axis: str = controls.get('x_axis', 'date')
+  y_axis: str = controls.get('y_axis', 'value')
+  round_num: int = controls.get('round_num', 19)
 
-  figure_control_values = dict(
-    scenarios=init_scenarios,
-    models=init_models,
-    location=init_location,
-    target=init_target,
+  figure_control_values = ChartControls(
+    x_axis=x_axis,
+    y_axis=y_axis,
+    round_num=round_num,
+    pathogen='covid',
+    scenario_ids=init_scenarios,
+    model_ids=init_models,
+    location_name=init_location,
     age_group=init_age_group,
-    uncertainty=init_uncertainty,
-    ensemble=init_ensemble,
+    target=init_target,
+    certainty_percent=init_uncertainty,
     annotations=init_annotations,
   )
+  chart = Chart(PlotType.LINE, figure_control_values)
 
   figure_container = html.Div(
     id='insight-visualization-figure',
-    children=chart(control_values=figure_control_values),
+    children=chart.get_fig(),
```

Comment:
i believe this is the source of that `Objects are not valid as a React child (found: object with keys {data, layout}).` error